Sources/ActionSheet/ActionSheetCell.swift

```diff
@@ -56,16 +63,33 @@ class ActionSheetCell: UICollectionViewCell {
         name.setContentHuggingPriority(.defaultLow, for: .horizontal)
         return name
     }()
-
-    let checkmark: UILabel = {
-        let checkmark = UILabel()
-        checkmark.text = "✓"
-        checkmark.font = UIFont.systemFont(ofSize: 18)
-        checkmark.textColor = PresentationTheme.current.colors.orangeUI
-        checkmark.translatesAutoresizingMaskIntoConstraints = false
-        checkmark.isHidden = true
-        return checkmark
+    
+    private var accessoryTypeImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        imageView.backgroundColor = .none
+        imageView.tintColor = PresentationTheme.current.colors.cellDetailTextColor
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
     }()
+    
+    var accessoryType: ActionSheetCellAccessoryType = .checkmark {
+        didSet {
+            switch accessoryType {
+            case .checkmark:
+                accessoryTypeImageView.image = UIImage(named: "checkmark")?.withRenderingMode(.alwaysTemplate)
+                accessoryTypeImageView.isHidden = !isSelected
+            case .disclosureChevron:
+                accessoryTypeImageView.image = UIImage(named: "disclosureChevron")?.withRenderingMode(.alwaysTemplate)
+                accessoryTypeImageView.isHidden = false
+            case .none:
+                accessoryTypeImageView.image = nil
+                accessoryTypeImageView.isHidden = true
+            default:
+                assertionFailure("Unhandled ActionSheetCellAccessoryType")
```

Comment:
Swift compiler complaining here.

Reply:
@carolanitz says its best to keep the default as an assertionFailure so that if we add more cases that we dont catch, it will assert fail

Reply:
In the case where we remove the default and add more cases to the enum without adding it to in the switch, Swift will most likely complain saying something like: `switch must be exhaustive...`

Reply:
Yeah that is true but also I also agree with Caro because although not entirely likely, what happens if the compiler is changed and that is removed. I don't have strong feelings either way, I as well dont like seeing the extra warnings so I can do either, if you have stronger feelings I dont mind changing it